email-auditing-api.yaml

```diff
@@ -0,0 +1,253 @@
+openapi: 3.0.1
+info:
+  description: |
+        Email Auditing API allows for downloading Invitation, Reminder, and/or Feedback (Rapid) response emails in a compressed file for specific Experience programs and Ad Hoc surveys within a defined period of time. By retaining the last version of these emails for up to 30 days, companies can manage their communication policies and comply with standard industry regulations.
+  title: Email Auditing API
+  version: 0.0.2
+servers:
+  - url: https://{api-gateway-hostname}
+    variables:
+      api-gateway-hostname:
+        default: instance-tenant.apis.medallia.com
+        description:
+          The Medallia Experience Cloud hostname used to access Medallia Audit Email API
+paths:
+  /audit-email/v0/emails/tarfile:
+    get:
+      description: |
+        Downloads Invitation, Reminder, and/or Feedback (Rapid) response emails in a compressed (.tar.gz) file within a set period of time.
+      operationId: downloadTar
+      parameters:
+        - description:
+            The oldest timestamp from which emails are downloaded. Timestamp is in second granularity and is inclusive (for example, 12:00:01 includes the first second of the minute). Timestamp must be in YYYY-MM-DDTHH:mm:ssZ format (ISO 8601).
+          in: query
+          name: since
+          required: true
+          schema:
+            format: date-time
+            type: string
+        - description:
+            The most recent timestamp from which emails are downloaded. Timestamp is exclusive (for example, 12:00:01 excludes the first second of the minute). Timestamp must be in YYYY-MM-DDTHH:mm:ssZ format (ISO 8601).
+          in: query
+          name: until
+          required: true
+          schema:
+            format: date-time
+            type: string
+        - description: List of the Experience programs or Ad Hoc surveys IDs (separated by comma) emails are part of.
+          in: query
+          name: program_ids
+          schema:
+            type: string
+      responses:
+        "200":
+          content:
+            application/octet-stream:
```

Comment:
The Content-Type returned by the API should be `application/gzip` to be in compliance with standards.

Reply:
Though Eng confirmed your suggestion as a specific Content-Type for gzip files, they're returning `application/octet-stream` :(